From stern@rowland.harvard.edu Fri Oct  5 11:43:07 2007
From: Alan Stern <stern@rowland.harvard.edu>
Date: Fri, 5 Oct 2007 14:43:00 -0400 (EDT)
Subject: USB: unusual_devs entry for Nikon DSC D2Xs
To: Greg KH <greg@kroah.com>, Phil Dibowitz <phil@ipom.com>
Cc: Paul Check <paul@openstreet.com>,  USB Storage list <usb-storage@lists.one-eyed-alien.net>
Message-ID: <Pine.LNX.4.44L0.0710051442050.5905-100000@iolanthe.rowland.org>


This patch (as996) adds an unusual_devs entry for the Nikon DSC D2Xs
camera.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Cc: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    7 +++++++
 1 file changed, 7 insertions(+)

--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -362,6 +362,13 @@ UNUSUAL_DEV(  0x04b0, 0x0413, 0x0110, 0x
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_FIX_CAPACITY),
 
+/* Reported by Paul Check <paul@openstreet.com> */
+UNUSUAL_DEV(  0x04b0, 0x0415, 0x0100, 0x0100,
+		"NIKON",
+		"NIKON DSC D2Xs",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_FIX_CAPACITY),
+
 /* BENQ DC5330
  * Reported by Manuel Fombuena <mfombuena@ya.com> and
  * Frank Copeland <fjc@thingy.apana.org.au> */
